src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  width: $card-size;
+  background: $color-main;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    @extend %font-default;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $card-h-color;
+  }
+
+  &__code {
+    @extend %font-default;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $card-p-color;
+
+    margin: 0;
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    @extend %font-default;
+
+    margin: 0;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    display: flex;
+    align-items: center;
+    color: $card-h-color;
+  }
+
+  &__price-block {
+    margin: 0;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 18px;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    @extend %font-default;
+
+    margin: 0;
+    font-weight: 400;
+    font-size: 12px;
+    color: $card-p-color;
+  }
+
+  &__price {
+    @extend %font-default;
+
+    margin: 0;
+    font-weight: 700;
+    font-size: 16px;
+    color: $card-h-color;
+  }
+
+  &__buy-button {
+    @extend %font-default;
+
+    margin: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 5px;
+    height: 40px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $color-main;
+    background-color: $color-secondary;
+
+    &:hover {
+      color: $color-secondary;
+      background-color: $color-main;
+      border: 1px solid $color-secondary;
```

Comment:
It's better to add a transparent border for the default state and just change its color on hover